packages/user-operation-controller/CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.0]
+
+### Changed
+
+- **BREAKING**: Add required `from` property to `PrepareUserOperationRequest` ([#3844](https://github.com/MetaMask/core/pull/3844))
+- **BREAKING**: Add required `from` property to `AddUserOperationRequest` ([#3844](https://github.com/MetaMask/core/pull/3844))
+- **BREAKING**: Make `smartContractAccount` optional in `AddUserOperationOptions` ([#3844](https://github.com/MetaMask/core/pull/3844))
+  - Use current account snap by default if not provided ([#3844](https://github.com/MetaMask/core/pull/3844))
+- **CHANGED**: Delete user operation if rejected during approval ([#3844](https://github.com/MetaMask/core/pull/3844))
```

Comment:
The changed prefix can be removed here since that formatting is specific to the PR description formatting.

Reply:
Right, done [here](https://github.com/MetaMask/core/pull/3864/commits/ff8f3f0bdb4c5485514f949c0e097950f1a5f6e6).